packages/core/styles/v2/components/icon.scss

```diff
@@ -1,5 +1,5 @@
 .cove-icon {
-  display: inline-flex;
+  display: block;
```

Comment:
fix: see the comment on modals

Reply:
Removed.  This was changed as a quick test to see if it had any effect on question mark styling.  It did not but I admit I forgot to remove it.